package.json

```diff
@@ -118,6 +118,11 @@
           "type": "boolean",
           "default": true,
           "description": "Enable/disable showing completions on potentially undefined values that insert an optional chain call. Requires TS 3.7+ and strict null checks to be enabled."
+        },
+        "angular.suggest.includeCompletionsWithSnippetText": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable snippet completions from angular language Server. Requires using TypeScript 4.3+ in the workspace."
```

Comment:
Is the part about requiring TS 4.3+ accurate? Wouldn't the feature simply require the client to have `snippetSupport` in the capabilities? Or have I forgotten something on the `@angular/language-service` side of things that requires TS 4.3?

This feature _does_ require that the "legacy View Engine" option is not enabled though (same with `includeAutomaticOptionalChainCompletions`, but I missed that in the last review)